### ISSUE TITLE
fix Tapable.plugin deprecation warning in webpack 4.x

### ISSUE THREAD

**Issue by xz64** [xz64/license-webpack-plugin#27]
```
DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
```

**Comment by xz64** [comment]
Fixed in v1.2.0

**Comment by xz64** [comment]
Initial fix for this broke consumers using webpack v3. Will need to spend more time seeing what went wrong.

**Comment by xz64** [comment]
Actually the fix for #26 was causing this. Closing this one and reopening the other.
